service/history/api/describeworkflow/api.go

```diff
@@ -162,13 +162,25 @@ func Invoke(
 			OriginalStartTime:       startEvent.EventTime,
 			CancelRequested:         executionInfo.CancelRequested,
 			ResetRunId:              executionInfo.ResetRunId,
+			RequestIdInfos:          make(map[string]*workflowpb.RequestIdInfo),
 		},
 	}
 
 	if mutableState.IsResetRun() {
 		result.WorkflowExtendedInfo.LastResetTime = executionState.StartTime
 	}
 
+	for requestID, requestIDInfo := range mutableState.GetExecutionState().GetRequestIds() {
+		info := &workflowpb.RequestIdInfo{
+			EventType: requestIDInfo.EventType,
+			Buffered:  requestIDInfo.EventId == common.BufferedEventID,
+		}
+		if !info.Buffered {
+			info.EventId = requestIDInfo.EventId
+		}
```

Comment:
Can we backfill event ID to `1` for `WorkflowExecutionStarted` `EventType`?

Reply:
For the `WorkflowExecutionStarted`, the request ID info will already have the correct ID from the start.